src/Symfony/Component/Console/Command/Command.php

```diff
@@ -356,6 +356,14 @@ public function complete(CompletionInput $input, CompletionSuggestions $suggesti
         }
     }
 
+    /**
+     * Get the callable from the InvokableCommand if available.
```

Comment:
`InvokableCommand` is internal, it should not be mentioned.


```suggestion
     * Get the code that is executed by the command.
```